docs/source/010_user.rst

```diff
@@ -4,7 +4,7 @@ User Guide for Lmod
 The guide here explains how to use modules. The User's tour of the
 module command covers the basic uses of modules. The other sections
 can be read at a later date as issues come up. The Advance user's
-guilde is for users needing to create their own modulefiles.
+guild is for users needing to create their own modulefiles. 
 
```

Comment:
This should be Guide, not guild, I think, as it continues "Advanced User's Guide" from the line above.